### DISCUSSION TITLE
Can other nerf models be converted to instant-ngp's snapshot msgpack file

### DISCUSSION THREAD

**Discussion by guijuzhejiang** [NVlabs/instant-ngp#963] [Q&A]
Thank you for your excellent work, the function of using the visual GUI to design the camera path is very nice. 
How can I convert the checkpoints trained by other nerf models into instant-ngp snapshot msgpack file to use the GUI to design the camera path. Is there a conversion script provided?

**Comment by Tom94** [comment]
No, unfortunately there is no simple way to convert other NeRF models to instant-ngp. It's not just a matter of the format, but also that other NeRF methods tend to use different data structures and neural network architectures, which are not amenable to interactive use.
